src/app/sign-up/sign-up.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { User } from '../interfaces';
+import { Router } from '@angular/router';
+import { TableService } from '../table-service';
+
+@Component({
+  selector: 'app-sign-up',
+  templateUrl: './sign-up.component.html',
+  styleUrls: ['./sign-up.component.scss'],
+})
+export class SignUpComponent implements OnInit {
+  user: User | any = {};
+  usersList: User[] = [];
```

Comment:
why we have usersList in register? 
Doing a register in facebook it loads all user data 😄  ?